### DISCUSSION TITLE
Can't build example on aarch64-linux

### DISCUSSION THREAD

**Discussion by AnthonyRonning** [rustshop/flakebox#127] [Q&A]
Running through the example and I got to this part: 

`flake.nix`
```
{
  description = "A very basic flake";

  inputs = {
    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";

  flakebox = {
    url = "github:rustshop/flakebox";
    inputs.nixpkgs.follows = "nixpkgs";
  };

    flake-utils.url = "github:numtide/flake-utils";
  };

  outputs = { self, nixpkgs, flakebox, flake-utils }:
    flake-utils.lib.eachDefaultSystem (system:
      let
        flakeboxLib = flakebox.lib.${system} { };
      in
      {
        devShells = flakeboxLib.mkShells {
          packages = [ ];
        };
      });
}
```

```
➜  flakebox-test git:(master) ✗ nix run nixpkgs#nix flake check
warning: Git tree '/Users/tony/Dev/Rust/learning/flakebox-test' is dirty
error: a 'aarch64-darwin' with features {} is required to build '/nix/store/ay9qys7hlr73j73h84s5saracxclck59-flakebox-flakebox-ci-yaml-gen.drv', but I am a 'aarch64-linux' with features {benchmark, big-parallel, gccarch-armv8-a, kvm, nixos-test}
```

The instructions say: `it means your nix is too old. Use nix run nixpkgs#nix flake check instead to run a newest version. With Nix even tools that are not currently installed are easy to use.`

But I am using that, so I'm not sure what else could be the problem.

**Comment by dpc** [comment]
Oh, I missed that issue. Looks like nix you get from nixpkgs is also too old. Possibly you need to update channel (`nix channel --update`) or something like that.

**Reply by AnthonyRonning** [reply]
Looks like I'm at an old version. Not wanting to do that right now as this is my main driver but I'll circle back to this. Thanks!
